conference.js

```diff
@@ -2599,13 +2613,24 @@ export default {
      * @returns {void}
      */
     _onConferenceJoined() {
+        const { dispatch } = APP.store;
+
         APP.UI.initConference();
 
         if (!config.disableShortcuts) {
             APP.keyboardshortcut.init();
         }
 
-        APP.store.dispatch(conferenceJoined(room));
+        dispatch(conferenceJoined(room));
+
+        const jwt = APP.store.getState()['features/base/jwt'];
+
+        if (jwt?.user?.hiddenFromRecorder) {
+            dispatch(muteLocal(true, MEDIA_TYPE.AUDIO));
+            dispatch(muteLocal(true, MEDIA_TYPE.VIDEO));
+            dispatch(setAudioUnmutePermissions(true, true));
+            dispatch(setVideoUnmutePermissions(true, true));
```

Comment:
Are we first going to add the tracks and mute them here? I'm wondering  if it is going to be better to do this earlier by imitating what we do for startAudioMuted and startVideoMuted. WDYT?

Reply:
I borrowed this code from @horymury 
https://github.com/jitsi/jitsi-meet/pull/10932/commits/11b5059c43aa20be17978ac3790babd52ae09e47
Not sure about it. Want me to change it in some other way?